README.md

```diff
@@ -6,14 +6,70 @@
 
 A collection of tools (including data analysis and machine learning applications) that can be used to explore radiation data (temporal and spectral) for external identification of radiation signatures.
 
-## Dependencies
+## Table of Contents
+
+1. [Installation](#Installation)
+
+2. [Usage](#Usage)
+
+## Installation
+
+### Dependencies
 
 * Python 3
 
-Including the following Python packages (found in `requirements.txt`):
+Versions 3.6-3.9 are currently supported by tests. The following Python packages are required to run (found in `requirements.txt`):
 
 * h5py
 * numpy
 * pandas
 * progressbar2
 * scipy
+
+Scripts can be run from the top-level directory:
+
+`python /path/to/RadClass/script`
+
+Or `RadClass` can be installed using pip:
+
+`python -m pip install /path/to/RadClass/. --user`
+
+## Usage
+
+RadClass is designed to be modular, allowing a user to use analysis modules as needed:
+
+<p align="center">
+  <img width="600" src="https://drive.google.com/uc?export=view&id=14RZCOaTmcwKaEjiWHaQeARaxCpx3zAur">
```

Comment:
Put the images in the repo, as well, rather than referring to gdrive 

Also, I think there is markdown syntax for embedding images
```
![GitHub Logo](/images/logo.png)
Format: ![Alt Text](url)
```